dataactvalidator/config/programActivityRules.csv

```diff
@@ -1,17 +1,6 @@
-SUM_FIELDS,FALSE,,DeobligationsRecoveriesRefundsdOfPriorYearByProgramObjectClass_CPE,"USSGL487100_DownwardAdjustmentsOfPriorYearUnpaidUndeliveredOrdersObligationsRecoveries_CPE, USSGL487200_DownwardAdjustmentsOfPriorYearPrepaidAdvancedUndeliveredOrdersObligationsRefundsCollected_CPE, USSGL497100_DownwardAdjustmentsOfPriorYearUnpaidDeliveredOrdersObligationsRecoveries_CPE,  USSGL497200_DownwardAdjustmentsOfPriorYearPaidDeliveredOrdersObligationsRefundsCollected_CPE",DeobligationsRecoveriesRefundsdOfPriorYearByProgramObjectClass_CPE = USSGL(4871+ 4872 + 4971 + 4972),B13,multi_field,B13
 EXISTS_IN_TABLE,TRUE,AllocationTransferAgencyIdentifier,"CGAC, cgac_code",3,Check that this is a valid agency code,A20_B,file_validation,A20
 EXISTS_IN_TABLE,TRUE,AgencyIdentifier,"CGAC, cgac_code",3,Check that this is a valid agency code,A20_B2,file_validation,A20
 REQUIRED_SET_CONDITIONAL,FALSE,,A16_B2,"ussgl480100_undeliveredordersobligationsunpaid_fyb, obligationsundeliveredordersunpaidtotal_fyb, ussgl490100_deliveredordersobligationsunpaid_fyb, obligationsdeliveredordersunpaidtotal_fyb, ussgl480200_undeliveredordersobligationsprepaidadvanced_fyb, grossoutlaysundeliveredordersprepaidtotal_fyb, ussgl490800_authorityoutlayednotyetdisbursed_fyb, grossoutlaysdeliveredorderspaidtotal_fyb, grossoutlayamountbyprogramobjectclass_fyb",FYB fields are required if reporting period end date is within Q1,A16_B,prerequisite,A16
-SET_EXISTS_IN_TABLE,FALSE,,ProgramActivity,"{""beginningperiodofavailability"":{""target_field"":""budget_year"",""skip_if_below"":2016}, ""agencyidentifier"":{""target_field"":""agency_id"",""pad_to_length"":3}, ""allocationtransferagencyidentifier"":{""target_field"":""allocation_transfer_id"",""pad_to_length"":3},""mainaccountcode"":{""target_field"":""account_number"",""pad_to_length"":4},""programactivitycode"":{""target_field"":""program_activity_code"",""pad_to_length"":4,""skip_if_blank"":true},""programactivityname"":{""target_field"":""program_activity_name"",""skip_if_blank"":true}}",Must be a valid program activity name and code for the corresponding TAS/TAFS as defined in Section 82 of OMB Circular A-11.,B9,multi_field,B9
+SET_EXISTS_IN_TABLE,FALSE,,ProgramActivity,"{""beginningperiodofavailability"":{""target_field"":""budget_year"",""skip_if_below"":2016}, ""agencyidentifier"":{""target_field"":""agency_id"",""pad_to_length"":3}, ""allocationtransferagencyidentifier"":{""target_field"":""allocation_transfer_id"",""pad_to_length"":3},""mainaccountcode"":{""target_field"":""account_number"",""pad_to_length"":4},""programactivitycode"":{""target_field"":""program_activity_code"",""pad_to_length"":4,""skip_if_blank"":true},""programactivityname"":{""target_field"":""program_activity_name"",""skip_if_blank"":true}}",Must be a valid program activity name and code for the corresponding TAS/TAFS as defined in Section 82 of OMB Circular A-11.,B9,multi_field,B9
```

Comment:
Was there a reason that we didn't include B9/B10 in this story?  Was this just an oversight when we listed the rules?